assets/dev/js/frontend/handlers/hotspot.js

```diff
@@ -0,0 +1,80 @@
+export default class Hotspot extends elementorModules.frontend.handlers.Base {
+    getDefaultSettings() {
+        return {
+            selectors: {
+                hotspot: '.elementor-hotspot',
+                hotspotTrigger: '.elementor-hotspot-trigger',
+                tooltip: '.elementor-tooltip',
+                tooltipMask: '.animation-direction-mask',
+            },
+        };
+    }
+
+    getDefaultElements() {
+        const selectors = this.getSettings( 'selectors' );
+        return {
+            $hotspot: this.$element.find( selectors.hotspot ),
+            $hotspotTrigger: this.$element.find( selectors.hotspotTrigger ),
+            $tooltip: this.$element.find( selectors.tooltip ),
+            $tooltipMask: this.$element.find( selectors.tooltipMask ),
+        };
+    }
+
+    bindEvents() {
+        const elementSettings = this.getElementSettings();
+        var tooltipTriggerEvent = 'mouseenter' === elementSettings.tooltip_trigger ? 'mouseleave mouseenter' : elementSettings.tooltip_trigger;
+
+        if ( tooltipTriggerEvent !== 'none' ) {
+            this.elements.$hotspot.on( tooltipTriggerEvent, this.onHotspotTriggerEvent.bind( this ) );
+        }
+    }
+
+    onHotspotTriggerEvent( event ) {
+        if ( jQuery( event.target ).is( '.elementor-hotspot-trigger' ) || jQuery( event.target ).parents( '.elementor-hotspot-trigger' ).length ) {
```

Comment:
I used the target check because I had to use the whole hotspot( hotspot-button and hotspot tooltip ) as a trigger event, specially for the hover event.
In the “mouse enter” event I need that the tooltip will not go away when I move the mouse from the button to the tooltip.
In the “mouse leave” event I need to run the handler-function also from tooltip and not only the hotspot-button.

I also found a bug and fix it, and also make the if statment more readable.
if ( isHotspotButtonEvent || isTooltipMouseLeave ) {.....  

If not clear I will explain.